frontend/src/static/js/v_authorship.js

```diff
@@ -39,7 +39,7 @@ window.vAuthorship = {
       totalLineCount: '',
       totalBlankLineCount: '',
       filesSortType: 'lineOfCode',
-      toReverseSortFiles: false,
+      toReverseSortFiles: 'descending',
```

Comment:
Maybe rename this variable name to `sortByOrder`.

Reply:
Yeah makes sense. Will do the needful.